[2021-09-27 15:51:30,965] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: day1_challenge2.echo_hello 2021-09-27T15:51:25.107283+00:00 [queued]>
[2021-09-27 15:51:31,040] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: day1_challenge2.echo_hello 2021-09-27T15:51:25.107283+00:00 [queued]>
[2021-09-27 15:51:31,044] {taskinstance.py:1095} INFO - 
--------------------------------------------------------------------------------
[2021-09-27 15:51:31,046] {taskinstance.py:1096} INFO - Starting attempt 1 of 1
[2021-09-27 15:51:31,048] {taskinstance.py:1097} INFO - 
--------------------------------------------------------------------------------
[2021-09-27 15:51:31,071] {taskinstance.py:1115} INFO - Executing <Task(BashOperator): echo_hello> on 2021-09-27T15:51:25.107283+00:00
[2021-09-27 15:51:31,081] {standard_task_runner.py:52} INFO - Started process 35694 to run task
[2021-09-27 15:51:31,094] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'day1_challenge2', 'echo_hello', '2021-09-27T15:51:25.107283+00:00', '--job-id', '99', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/day1_challenge2.py', '--cfg-path', '/tmp/tmpy68jojt6', '--error-file', '/tmp/tmpzv5jypc7']
[2021-09-27 15:51:31,099] {standard_task_runner.py:77} INFO - Job 99: Subtask echo_hello
[2021-09-27 15:51:31,429] {logging_mixin.py:109} INFO - Running <TaskInstance: day1_challenge2.echo_hello 2021-09-27T15:51:25.107283+00:00 [running]> on host 196580edc338
[2021-09-27 15:51:31,691] {taskinstance.py:1254} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Attasil Chitwirote
AIRFLOW_CTX_DAG_ID=day1_challenge2
AIRFLOW_CTX_TASK_ID=echo_hello
AIRFLOW_CTX_EXECUTION_DATE=2021-09-27T15:51:25.107283+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-09-27T15:51:25.107283+00:00
[2021-09-27 15:51:31,712] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2021-09-27 15:51:31,763] {subprocess.py:63} INFO - Running command: ['bash', '-c', "echo 'hello'"]
[2021-09-27 15:51:31,866] {subprocess.py:74} INFO - Output:
[2021-09-27 15:51:31,889] {subprocess.py:78} INFO - hello
[2021-09-27 15:51:31,933] {subprocess.py:82} INFO - Command exited with return code 0
[2021-09-27 15:51:32,133] {taskinstance.py:1219} INFO - Marking task as SUCCESS. dag_id=day1_challenge2, task_id=echo_hello, execution_date=20210927T155125, start_date=20210927T155130, end_date=20210927T155132
[2021-09-27 15:51:32,204] {local_task_job.py:151} INFO - Task exited with return code 0
[2021-09-27 15:51:32,288] {local_task_job.py:261} INFO - 1 downstream tasks scheduled from follow-on schedule check
